### ISSUE TITLE
Pretrained LayoutLM model results in different architecture

### ISSUE THREAD

**Issue by spencer-hong** [doc-analysis/DocBank#32]
Hi, I'm trying to replicate your results, and I see that the config.json for the pretrained LayoutLM has "bert" as the the model type.

When I load the pretrained model, it results in `You are using a model of type bert to instantiate a model of type layoutlm. This is not supported for all configurations of models and can yield errors.`

I was able to replicate your results for Roberta-Large and Bert-Large, but not for LayoutLM. Could you advise? Thank you.

**Comment by 5cp** [comment]
Hi @spencer-hong 

I believe you need to use the LayoutlmModel class provided in the LayoutLM repo, rather than trying to load the Docbank-provided pretrained weights using the Huggingface LayoutLMForTokenClassification class.

ex:

```
from unilm.layoutlm.layoutlm.modeling.layoutlm import LayoutlmModel
model = LayoutlmModel.from_pretrained("./layoutlm_base_500k_epoch_1", num_labels=12)
```

**Comment by spencer-hong** [comment]
Hi @5cp, I was able to replicate the LayoutLM results and reached 92% accuracy on the validation set. Thanks so much for your help.
